pkg/dockerclient/shell_test.go

```diff
@@ -226,6 +233,54 @@ func (suite *ShellClientTestSuite) TestBuildRetryOnErrors() {
 	suite.mockedCmdRunner.AssertNumberOfCalls(suite.T(), "Run", 4)
 }
 
+func (suite *ShellClientTestSuite) TestBuildFailValidation() {
+
+	for _, buildOptions := range []BuildOptions{
```

Comment:
consider use testCase pattern with `name` field to allow running each test with its own scope
see `pkg/common/helper_test.go:417` for an example